src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss

```diff
@@ -1,12 +1,13 @@
-//
-// ===== Mixins for <hx-checkbox> facade states =====
-//
+// ================================================== //
+// ===== Mixins for <hx-checkbox> facade states ===== //
+// ================================================== //
 
 // ========== PRISTINE ==========
 @mixin __checkboxFacade($pseudo-state: null) {
   @if $pseudo-state == focus {
+    border-color: $blue-900;
     border-color: var(--hxCheckbox-bordercolor-focus, $blue-900);
-    box-shadow: $focus-glow;
+    // box-shadow: $focus-glow; // TBD v2.0 specs
```

Comment:
again let me know if we have a straight answer around box shadows across the design system

Reply:
Yes, I'll check with @danielmdesigns on his recommendations for checkbox.